docs/testing.md

```diff
@@ -1,5 +1,13 @@
 # Writing Tests
 
+## Controlling test verbosity
+
+To run tests verbosely (displaying test events and output and error streams to the console), use the following system property:
+
+```shell
+./gradlew test -PverboseTest
```

Comment:
As I said in #994 I think it would be better to have verbose tests by default and then provide a flag to silence them, especially because otherwise it will be tricky on the IDE part having to set every time this flag.

Reply:
@paullatzelsperger I don't have an opinion on this - since you wrote the story, what's your take?

Reply:
@ndr-brt I am of the opposite opinion: make them as sparse as possible by default, so that they don't needlessly spam the terminal window, and on CI builds we can enable some "verbosity" flag

Reply:
Can we go ahead and merge it as is, since it is already the current behavior? We can create an issue to change the setting to verbose and collect opinions

Reply:
@paullatzelsperger there are two way of using tests (on my side):
- from IDE: probably launching one test or one test class most of the time, if the test is broken, some logs would be appreciated for sure, if everytime I launch a different test from the IDE I have to modify the "run configuration" and add the flag, it could become annoying.
- from command line: at the end before the commit to verify that everything works correctly, probably here logs are not wanted, and would be straightforward to add a flag to shut them down, same on the CI.

When we introduced the system tests some other devs were asking me why there weren't logs printed by default.

Reply:
Created #1049 to capture this proposal. We should please go forward with this PR that solidifies the current behavior and ensures we have proper reporting of failing tests in CI. We can take the discussion to #1049 and reach an agreement for another PR.

Reply:
@ndr-brt ok to approve?